facturacion_mexico/one_offs/cleanup_mop_canonical.py

```
@@ -0,0 +1,115 @@
+    # 4. Reporte — no pueden borrarse
+    print(f"\n{'─'*60}")
+    print(f"⛔ CON REFERENCIAS — no se borrarán: {len(cannot_delete)}\n")
+    for r in cannot_delete:
+        print(f"   • {r['name']}")
+        for ref_field, count in r["refs"].items():
+            print(f"     └─ {ref_field}: {count} documento(s)")
+
+    print(f"\n{'─'*60}")
+
+    if dry_run:
+        print("ℹ️  MODO DRY RUN — no se eliminó nada.")
```

Ruff (0.15.15)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)